src/openkit-shared/Protocol/HTTPClientWebClient.cs

```diff
@@ -64,11 +65,18 @@ private static HTTPResponse ReadResponse(System.Net.WebResponse webResponse)
                 response = reader.ReadToEnd();
             }
             System.Net.HttpStatusCode responseCode = ((System.Net.HttpWebResponse)webResponse).StatusCode;
+            Dictionary<string, List<string>> headers = new Dictionary<string, List<string>>();
+            for (var i = 0; i < webResponse.Headers.Count; i++)

```

Comment:
is it possible to iterate over the key list of webResponse.Headers and directly use the element to retrieve the according values in .GetValues ?
It would be possible to remove line 71 in that case.